src/site/content/en/progressive-web-apps/pwa-checklist/index.md

```diff
@@ -0,0 +1,482 @@
+experience with JavaScript to do form validation and submit the form via
+AJAX, improving the experience for users who can support it.
+
+Consider that your users will experience your site across a spectrum of
+devices and browsers. You cannot simply target the top end of the spectrum.
+By using feature detection, you'll be able to deliver a usable experience
+for the widest range of potential users, including those using browsers
+and devices that may not exist today.
+
+#### How
+
+Jeremy Keith's [Resilient Web Design](https://resilientwebdesign.com/) is an
+excellent resource describing how to think about web design in this
+cross-browser, progressive methodology.
+
+#### Additional reading
+
+* A List Apart's [Understanding Progressive Enhancement](https://alistapart.com/article/understandingprogressiveenhancement/)
+  is a good introduction to the topic.
+* Smashing Magazine's
+  [Progressive Enhancement: What It Is, And How To Use It?](https://www.smashingmagazine.com/2009/04/progressive-enhancement-what-it-is-and-how-to-use-it/)
+  gives a practical introduction and links to more advanced topics.
+* MDN has an article titled
+  [Implementing feature detection](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection)
+  that talks about how to detect a feature  by directly querying it.
+
+{% endDetails %}
+
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+
+Responsive to any screen size
+
+Users can use your PWA on any screen size and all of the content is available
+at any viewport size.
+
+{% endDetailsSummary %}
+
+Users can use your PWA on any screen size and all of the content is available
+at any viewport size.
+
+#### Why
+
+Devices come in a range of sizes, and users may use your application at a
+range of sizes, even on the same device. Therefore, it's critical to ensure
+your content not just fits within the viewport, but that all features and
+content for your site are usable at all viewport sizes.
+
+The tasks users want to complete and the content they want to access do
+not change with viewport size. The content can be rearranged at different
+viewport sizes, and it should all be there, one way or another. In fact, as
+Luke Wroblewski states in his book Mobile First, starting small and going
+large instead of the other way around can actually improve a site's design:
+
+> Mobile devices require software development teams to focus on only the
+> most important data and actions in an application. There simply isn't room
+> in a 320 by 480 pixel screen for extraneous, unnecessary elements.
+> You have to prioritize.
+>
+> So when a team designs Mobile First, the end result is an experience focused
+> on the key tasks users want to accomplish without the extraneous detours
+> and general interface debris that litter today's desktop-accessed Web sites.
+> That's good user experience and good for business.
+
+#### How
+
+There are many  resources on responsive design, including the
```

Comment:
This could be a list as could much of the content in this piece. Considering this is a reference and not an article, bullets throughout would increase scanability and make some content function like check lists. (The latter doesn't apply to this paragraph, obviously.)